portfolio.js

```diff
@@ -0,0 +1,18 @@
+var projects = [];
+
+function Project () {
+  this.title = title;
+  this.dateCreated = dateCreated;
+  this.image = image;
+  this.fileSource = fileSource;
+  projects.push(this);
+}
+
+Project.prototype.toHtml = function() {
```

Comment:
Great work adding this method to your constructor and creating your template element!